plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts

```diff
@@ -814,10 +814,6 @@ describe('KubernetesProxy', () => {
         ),
       );
 
-      // Prepopulate the proxy so the WebSocket upgrade can happen, result doesn't actually matter
-      const result = await fetch(wsProxyAddress.replace('ws', 'http'));
```

Comment:
Since you're not using `node-fetch` anymore you've gotta remove the import in this file

Reply:
Missed it, will fix